### PR TITLE
Add stress testing scripts

### PR DESCRIPTION
**Pull request by ehlertjd**
See [README.md](https://github.com/flywheel-io/core/blob/2ac2a12a83e6f746f764e798583c5b4c3c55958e/tests/stress_tests/README.md) 

These are scripts to stress test loading the flywheel hierarchy via SDK. In the case of python also do some basic validation, and sample file downloads.

### Test Requirements

- NONE? 

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

File: tests/stress_tests/python/walk.py [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+        self.logger = logger
+        self.container_type = container_type
+        self.container = container
+
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, traceback):
+        if exc_type == ApiException:
+            print('Caught ApiException: {}'.format(exc_val), file=sys.stderr)
```
Comment (hkethi002) [inline tests/stress_tests/python/walk.py]: Doesn't work with python 2.7
```
  File "tests/stress_tests/python/walk.py", line 26
    print('Caught ApiException: {}'.format(exc_val), file=sys.stderr)
                                                         ^
```

**Comment by hkethi002** [comment]
That was meant to be changes requested

File: tests/stress_tests/python/walk.py (hunk 1 above)
Comment (ehlertjd) [reply tests/stress_tests/python/walk.py]: Added comment in addition to the [hashbang](https://github.com/flywheel-io/core/pull/1309/files/2ac2a12a83e6f746f764e798583c5b4c3c55958e#diff-905a5ea190d252f042d7a101501c299dR1) line.

**Comment by nagem** [comment]
I looked over it and didn't see any issues. Definitely safe for a merge 👍 

Thanks for putting this together, we should look into ways to automate running it before releases.

**Pull request merged**